prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm_states.c

```diff
@@ -1634,9 +1634,6 @@ ssize_t fi_ibv_rdm_req_hndls_init(void)
 	fi_ibv_rdm_req_hndl_arr[FI_IBV_STATE_EAGER_READY_TO_FREE]
 		[FI_IBV_STATE_RNDV_RECV_END][FI_IBV_EVENT_POST_LC] =
 			fi_ibv_rdm_rndv_recv_ack_lc;
-	fi_ibv_rdm_req_hndl_arr[FI_IBV_STATE_EAGER_RECV_END]
```

Comment:
The handler is the same as the following:

https://github.com/ofiwg/libfabric/blob/2f07e4f08eb371de16880fb26ce4bee92c6ca909/prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm_states.c#L1618